test/e2e/csv_e2e_test.go

```diff
@@ -80,7 +80,7 @@ var _ = Describe("ClusterServiceVersion", func() {
 					TargetNamespaces: []string{ns.GetName()},
 				},
 			}
-			Expect(ctx.Ctx().Client().Create(context.TODO(), &og)).To(Succeed())
+			Eventually(Expect(ctx.Ctx().Client().Create(context.TODO(), &og)).To(Succeed()))
```

Comment:
we should probably switch to `context.Background()` in these tests (check out the godocs).